.gitignore

```diff
@@ -12,7 +12,7 @@ coverage
 package-lock.json
 
 # production
-dist
+# dist
```

Comment:
Remove dist folder before merging to master

Reply:
Yes